content/en/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -313,16 +313,13 @@ limit, the same as the scheduler.
 A Pod can restart, causing re-execution of init containers, for the following
 reasons:
 
-* A user updates the Pod specification, causing the init container image to change.
-  Any changes to the init container image restarts the Pod. App container image
-  changes only restart the app container.
 * The Pod infrastructure container is restarted. This is uncommon and would
   have to be done by someone with root access to nodes.
-* All containers in a Pod are terminated while `restartPolicy` is set to Always,
-  forcing a restart, and the init container completion record has been lost due
-  to garbage collection.
```

Comment:
Isn't this still true?

Reply:
Let me test it again/
/hold

Reply:
/hold cancel
@tengqm 
You are right, this is still true.
I add it back now.